staticd/static_vty.c

```diff
@@ -431,10 +431,10 @@ static int static_route_leak(
 		    || strncasecmp(ifname, "blackhole", strlen(ifname)) == 0) {
 			if (vty)
 				vty_out(vty,
-					"%% Nexthop interface cannot be Null0, reject or blackhole\n");
+					"%% Null0, reject and blackhole are reserved keywords. Cannot use a nexthop interface with a reserved name.\n");
```

Comment:
code changes looks good, but error message may be long. 

what about 
"Nexthop interface name can not be from reserved keywords (Null0, reject, blackhole)"